src/translations/en.json

```diff
@@ -396,6 +396,7 @@
   "InquiryForm.messageLabel": "Message to {authorDisplayName}",
   "InquiryForm.messagePlaceholder": "Hello {authorDisplayName}! I'm looking forward to…",
   "InquiryForm.messageRequired": "A message is required",
+  "InquiryForm.noTransactionRightsError": "Oops, something went wrong. You don't have permission to make inquiries. <NoAccessLink>Read more about transaction rights.</NoAccessLink>",
```

Comment:
I'm not sure if Katri was asked to review this text and I find it a bit odd (even if accurate) that it says "inquiries". I would just use the same "You don't have transaction rights" here, to use fewer concepts and not force end-users to have to understand what is an inquiry and what is a transaction.

Reply:
If/When this is changed, then I think the related message should also changed:
`"InquiryForm.userPendingApprovalError": "Oops, something went wrong. You don't have permission to make inquiries",
`
Although, that message is only shown, if we or customizers create a bug that reveals the modal (or customizers embed the form directly to the listing page without a modal).

Reply:
I've now changed these two marketplace texts to 

```
"InquiryForm.noTransactionRightsError": "Oops, something went wrong. You don't have transaction rights. <NoAccessLink>Read more about transaction rights.</NoAccessLink>",
"InquiryForm.userPendingApprovalError": "Oops, something went wrong. Your account is waiting for approval.",
```